src/dl_convert.cpp

```diff
@@ -6,23 +6,6 @@
 #include <dl/dl.h>
 #include <dl/dl_convert.h>
 
-template <typename T, int SIZE>
-class CArrayStatic
```

Comment:
Made this growable.
Moved it to "src/dl_types.h". Perhaps there is a better place?
Should it be renamed?

Reply:
It should probably be renamed but as it's an "implementation detail" not affecting user code I think you could ignore that for now.